packages/fxa-payments-server/src/index.scss

```diff
@@ -47,26 +47,6 @@ body {
   }
 }
 
-.main-content {
-  display: grid;
-  grid-template-columns: calc(100vw - 24px);
-  grid-template-rows: min-content;
-  grid-column-gap: 16px;
-  margin-top: $header-height;
```

Comment:
At the moment I've hardcoded the `$header-height` values. How have we handled these in other tailwind changes?

Reply:
We've hardcoded these values as well :)